internal/ykclient/api_handlers.go

```diff
@@ -0,0 +1,296 @@
+package ykclient
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/apache/yunikorn-core/pkg/webservice/dao"
+)
+
+var (
+	streamEndPt            = "/ws/v1/events/stream"
```

Comment:
Maybe eventually we should move these into shared definitions in some place like a `internal/config/constants.go` or some such? The `yunikorn-k8shim` repo puts lots of endpoints in its `test/e2e/framework/configmanager/constants.go`, for example.